xsd/netex_part_1/part1_tacticalPlanning/netex_passengerInformationEquipment_version.xsd

```diff
@@ -243,10 +242,15 @@ LOGICAL DISPLAY corresponds to a SIRI STOP MONITORING point.</xsd:documentation>
 			<xsd:element ref="LogicalDisplayRef" minOccurs="0"/>
 			<xsd:element ref="StopPlaceRef" minOccurs="0"/>
 			<xsd:element ref="SiteComponentRef" minOccurs="0"/>
+			<xsd:element name="PassengerInformationEquipmentList" type="PassengerInformationEquipmentEnumeration" minOccurs="0">
```

Comment:
1) Do you really want to allow several values? If yes, then it would be better to have a ListOfPassengerInformationEquipment like at other places where we have several values.
Eg. instead of 
<PassengerInformationEquipmentList>DepartureMonitor</PassengerInformationEquipmentList>
<PassengerInformationEquipmentList>AudioAnnouncement</PassengerInformationEquipmentList>
you would have
<PassengerInformationEquipmentList>DepartureMonitor AudioAnnouncement</PassengerInformationEquipmentList>

Reply:
Oh I see the comments box removes the xml tags

Reply:
I am open to both. @Aurige should I add only one Enum?